resources/deployments/controller/controller_deployments_external_test.go

```diff
@@ -44,6 +46,16 @@ func makeDeviceAuthHeader(claim string) string {
 		base64.StdEncoding.EncodeToString([]byte(claim)))
 }
 
+func makeApi(router rest.App) *rest.Api {
+	api := rest.NewApi()
+	api.Use(
+		&requestlog.RequestLogMiddleware{},
```

Comment:
ok, your call. I need to refactor the middleware then:
- right now it derives from a global logger
- I'll make it accept a base logger, which will override the global one if the user provides it

then the user will be able to pass the root/reference logger with whatever config they please.

I just need an ACK that we're on the same page before I start.